be1-go/message/query/method/answer.go

```diff
@@ -0,0 +1,17 @@
+package method
```

Comment:
All files in the method folder are in the package method. It would make sense to change it, but I think it is better to change it in another PR.

Reply:
Sorry I still don't get why you are redefining an `Answer` type, while there is an existing one in `message/answer.go`.

Reply:
I didn't understand what you were asking at first. So yeah, there is no specific reason, I guess I just missde the class. And it could (and should) be done (I already looked at it a bit), I just have to find a way to rewrite the tests as they are posing me a few problems because of the json schema validation. Would it be better to do it now, or to write an issue that I will work on later ?

Reply:
If its not a huge work then please include that in this PR.